### PR TITLE
Update install.rst

### LINKED ISSUE #683: ERROR: Could not build wheels for amici which use PEP 517 and cannot be installed directly
Hi developers,

this might be an easy one to fix:

I'm facing the following error message when executing `tox -e base`:
`Could not build wheels for amici which use PEP 517 and cannot be installed directly.`

I followed the steps described on  https://pypesto.readthedocs.io/en/latest/contribute.html  (I am on the develop branch).

Cheers!

### PR DESCRIPTION
**Pull request by PaulJonasJost**
fixes #683

### CODE REVIEW

File: doc/install.rst [hunk 1]
```diff
@@ -61,17 +61,23 @@ If you want to upgrade from an existing previous version, replace
...
+  Or follow the instructions from the
+  `AMICI github <https://github.com/AMICI-dev/AMICI/blob/master/INSTALL.md>`_.
 
 * This package inherently supports optimization using the dlib toolbox.
   To use it, install dlib via::
 
    pip3 install dlib
+   
+* All external dependecies can be installed through
+  `this shell script <https://github.com/AMICI-dev/AMICI/blob/master/INSTALL.md>`_.
```
Comment (dweindl) [inline doc/install.rst:83]: This is not a shell script :)

File: doc/install.rst [hunk 2]
```diff
@@ -61,17 +61,23 @@ If you want to upgrade from an existing previous version, replace
 ``install`` by ``ìnstall --upgrade`` in the above commands.
 
 
-Install optional packages
--------------------------
+Install optional packages and external dependencies
+---------------------------------------------------
 
 * This package includes multiple comfort methods simplyfing its use for
```
Comment (dweindl) [inline doc/install.rst:67]:
```suggestion
* This package includes multiple comfort methods simplifying its use for
```

File: doc/install.rst (hunk 1 above)
Comment (PaulJonasJost) [reply doc/install.rst:83]: ohhh wrong link :D thanks will change it 👍

File: doc/install.rst [hunk 3]
```diff
@@ -61,17 +61,23 @@ If you want to upgrade from an existing previous version, replace
...
   parameter estimation for models generated using the toolbox
   `amici <https://www.github.com/icb-dcm/amici>`_.
```
Comment (dweindl) [inline doc/install.rst:69]:
```suggestion
  `AMICI <https://github.com/AMICI-dev/AMICI>`_.
```

File: doc/install.rst (hunk 1 above)
Comment (PaulJonasJost) [inline doc/install.rst:83]:
```suggestion
  `this shell script < https://github.com/ICB-DCM/pyPESTO/blob/main/.github/workflows/install_deps.sh>`_.
```

File: doc/install.rst [hunk 4]
```diff
@@ -61,17 +61,23 @@ If you want to upgrade from an existing previous version, replace
...
   To use AMICI, install it via pip::
 
     pip3 install amici
+    
+  Or follow the instructions from the
+  `AMICI github <https://github.com/AMICI-dev/AMICI/blob/master/INSTALL.md>`_.
```
Comment (dweindl) [inline doc/install.rst:75]:
```suggestion
  `AMICI documentation <https://amici.readthedocs.io/en/latest/python_installation.html>`_.
```

File: doc/install.rst [hunk 5]
```diff
@@ -61,17 +61,23 @@ If you want to upgrade from an existing previous version, replace
...
 
 * This package inherently supports optimization using the dlib toolbox.
   To use it, install dlib via::
 
    pip3 install dlib
+   
+* All external dependecies can be installed through
+  `this shell script < https://github.com/ICB-DCM/pyPESTO/blob/main/.github/workflows/install_deps.sh>`_.
```
Comment (PaulJonasJost) [inline doc/install.rst:83]:
```suggestion
  `this shell script < https://github.com/ICB-DCM/pyPESTO/blob/main/.github/workflows/install_deps.sh>`_.

```

Comment (dweindl) [inline doc/install.rst:83]:
```suggestion
  `this shell script <https://github.com/ICB-DCM/pyPESTO/blob/main/.github/workflows/install_deps.sh>`_.
```

File: doc/install.rst [hunk 6]
```diff
@@ -61,17 +61,23 @@ If you want to upgrade from an existing previous version, replace
 ``install`` by ``ìnstall --upgrade`` in the above commands.
 
 
-Install optional packages
--------------------------
+Install optional packages and external dependencies
+---------------------------------------------------
 
-* This package includes multiple comfort methods simplyfing its use for
+* This package includes multiple comfort methods simplifying its use for
```
Comment (dilpath) [inline doc/install.rst:67]:
```suggestion
* pyPESTO includes multiple convenience methods to simplify
```

Usually "convenience functions", though perhaps "convenience methods" or "helper methods" is more appropriate for Python.

File: doc/install.rst [hunk 7]
```diff
@@ -61,17 +61,23 @@ If you want to upgrade from an existing previous version, replace
...
 
+  Or follow the instructions from the
+  `AMICI documentation <https://amici.readthedocs.io/en/latest/python_installation.html>`_.
+
 * This package inherently supports optimization using the dlib toolbox.
   To use it, install dlib via::
 
    pip3 install dlib
+
+* All external dependecies can be installed through
```
Comment (dilpath) [inline doc/install.rst:82]:
```suggestion
* All external dependencies can be installed through
```

File: doc/install.rst [hunk 8]
```diff
@@ -61,17 +61,23 @@ If you want to upgrade from an existing previous version, replace
...
-* This package includes multiple comfort methods simplyfing its use for
+* This package includes multiple comfort methods simplifying its use for
   parameter estimation for models generated using the toolbox
-  `amici <https://www.github.com/icb-dcm/amici>`_.
+  `AMICI <https://github.com/AMICI-dev/AMICI>`_.
   To use AMICI, install it via pip::
 
     pip3 install amici
 
+  Or follow the instructions from the
```
Comment (dilpath) [inline doc/install.rst:74]:
```suggestion
  or follow the instructions from the
```

**Review by dilpath — COMMENTED** [review]
- "PIP" in a title can be "Pip" or `pip`
- "GIT" in a title can be "Git" or `git`
- "ENV_NAME" can be `ENV_NAME`
- Is the requirement now `Python>=3.7.1` due to NumPy?

File: doc/install.rst [hunk 9]
```diff
@@ -61,17 +61,23 @@ If you want to upgrade from an existing previous version, replace
...
-* This package includes multiple comfort methods simplyfing its use for
+* pyPESTO includes multiple convenience methods to simplify
   parameter estimation for models generated using the toolbox
-  `amici <https://www.github.com/icb-dcm/amici>`_.
+  `AMICI <https://github.com/AMICI-dev/AMICI>`_.
   To use AMICI, install it via pip::
 
     pip3 install amici
 
+  or follow the instructions from the
```
Comment (yannikschaelte) [inline doc/install.rst:74]:
```suggestion
  or, in case of problems, follow the full instructions from the
```

**Review by yannikschaelte — APPROVED** [review]
thanks!

**Pull request merged**